# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bsd-games/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Maybe not really gcc-4.3, but even earlier version. In any case recent GCC's
do not like multiple parameters named equally:

dab/gamescreen.h:73: error: multiple parameters named 'p'

  - Rene Rebe <rene@exactcode.de>

--- bsd-games-2.17/dab/gamescreen.h.vanilla	2008-08-12 11:34:50.000000000 +0200
+++ bsd-games-2.17/dab/gamescreen.h	2008-08-12 11:35:00.000000000 +0200
@@ -70,9 +70,9 @@
     virtual void redraw(void) = 0;			// Refresh
     virtual int getinput(void) = 0;			// Get user input
     virtual void bell(void) = 0;			// Beep
-    virtual void score(size_t p, const PLAYER& p) = 0;	// Post current score
-    virtual void games(size_t p, const PLAYER& p) = 0;	// Post games won
-    virtual void total(size_t p, const PLAYER& p) = 0;	// Post total score
+    virtual void score(size_t p, const PLAYER& pl) = 0;	// Post current score
+    virtual void games(size_t p, const PLAYER& pl) = 0;	// Post games won
+    virtual void total(size_t p, const PLAYER& pl) = 0;	// Post total score
     virtual void ties(const PLAYER& p) = 0;		// Post tie games
 };
 
